event_sale_confirmation/models/event_registration.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017,2018 IT-Projects LLC - Ivan Yelizariev
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+from odoo import models, fields, api
+
+
+class EventRegistration(models.Model):
+
+    _inherit = 'event.registration'
+
+    event_ticket_price = fields.Float(
+        related='event_ticket_id.price',
+        readonly=True)
+    user_is_manager = fields.Boolean(compute='_compute_user_is_manager')
```

Comment:
I'm not sure if something like `can_confirm` would be a more appropiate name for this field

Reply:
It's not so. The user can confirm when he is a manager OR ticket is free

Reply:
True that :+1: